integration_tests/__init__.py

```diff
@@ -111,3 +112,9 @@ def logout(self):
         expected = ('Clearing credentials for Ubuntu One SSO.\n'
                     'Credentials cleared.\n')
         self.assertEqual(expected, output)
+
+    def register(self, snap_name):
+        self.run_snapcraft(['register', snap_name])
+        # sleep a few seconds to avoid hitting the store restriction on
+        # following registrations.
+        time.sleep(10)
```

Comment:
For the record, Store staging has 5 s rate-limit and production 60 s, a comment on the test would not hurt.